pkg/controller/utils/capability.go

```diff
@@ -330,3 +335,16 @@ func fixOpenAPISchema(name string, schema *openapi3.Schema) {
 	}
 	schema.Description = description
 }
+
+// ConvertOpenAPISchema2SwaggerObject converts OpenAPI v2 JSON schema to Swagger Object
+func ConvertOpenAPISchema2SwaggerObject(data []byte) (*openapi3.Schema, error) {
+	swagger, err := openapi3.NewSwaggerLoader().LoadSwaggerFromData(data)
+	if err != nil {
+		return nil, err
+	}
+	schemaRef := swagger.Components.Schemas["parameter"]
```

Comment:
use a const variable for `parameter`, maybe we already have one in the project

Reply:
Done, replaced the string all over the project.